app/src/main/java/pl/droidevs/books/library/ExportImportViewModel.java

```diff
@@ -0,0 +1,97 @@
+package pl.droidevs.books.library;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.Observer;
+import android.arch.lifecycle.ViewModel;
+import android.os.Environment;
+import android.support.annotation.Nullable;
+
+import org.supercsv.io.CsvBeanWriter;
+import org.supercsv.prefs.CsvPreference;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import pl.droidevs.books.entity.BookEntity;
+import pl.droidevs.books.repository.BookCsvRepository;
+
+public class ExportImportViewModel extends ViewModel {
+
+    private static final String FILE_NAME = "Books.csv";
+    private BookCsvRepository bookCsvRepository;
+
+    @Inject
+    public ExportImportViewModel(BookCsvRepository bookRepository) {
+        this.bookCsvRepository = bookRepository;
+    }
+
+    public boolean exportBooks() {
```

Comment:
Zwracanie flagi jako informację o błędzie nie jest najlepszą parktyką. Co mówi mi wywołanie if (exportBooks())? Czy true oznacza, że są książki do eskportu? Czy że można eksportować książki? A może, że udało się wyeksportować? Nie mogę jednoznacznie odpowiedzieć na to pytanie bez zaglądania do kodu tej metody.